articles/what-is-ransomware/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/what-is-ransomware/
+title: What is Ransomware?
+description: Ransmoware is a type of cybersecurity threat, or malware (malicious software) that attackers use to hack into their victims' devices, and force them to pay a ransom.
+author: maitreyi-karanjkar
+date: 2020-10-09T00:00:00-07:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/what-is-ransomware/hero.jpg
+    alt: ransomware
+---
+In a world where computers are an essential part of life, it is only reasonable to have a healthy fear of cyber security threats. Cyber attacks are perpetually happening, and their targets can range from big corporations, to national governments, and even to regular people––potentially with their livelihoods at stake, depending on the nature of the threat.
```

Comment:
Needs quantification.

Introduction is very vague and pertains to knowledge that is common sense. It might be helpful to add a section to contextualize the big threats you are referring to. See: https://usa.kaspersky.com/resource-center/threats/ransomware-examples for details on examples of ransomware

When you say "cyber attacks are perpetually happening," this is unsubstantiated. Please use external sources to inform the reader as to how many, who they affect most = use data sources and citations.